earthly/flutter/Earthfile

```diff
@@ -52,6 +52,7 @@ flutter-base:
     RUN dart pub global activate melos
     RUN dart pub global activate junitreport
     RUN dart pub global activate coverage
+    RUN dart pub global activate combine_coverage
```

Comment:
where will this be used? do you intend to add the usage in this PR or somewhere else?

Reply:
I'll use it in this PR https://github.com/input-output-hk/catalyst-voices/pull/661